### ISSUE TITLE
Incorrect reporting of ERC `last_time` for v11.3.0.x

### ISSUE THREAD

**Issue by indistinctTalk** [andywalden/esmcheckds2#28]
I suspect this is a bug or API change over in McAfee-land but the reporting of ERC `last_time` values via the API in v11.3.0, up to and including patch 13, appears to be broken.

I see the below when running via a shell. Before anyone goes scrolling way off to the right, the relevant value is `'last_time': '21-07-2020 01:58:06'`.

```python
>>> esm = ESM(config)
>>> dtree = DevTree(esm)
>>> ercs = [d for d in dtree.siem_devices() if d['desc_id'] == '2']
>>> ercs[0]
{'idx': 2, 'desc_id': '2', 'name': 'xxx', 'ds_id': 'xxx', 'enabled': 'F', 'ds_ip': 'xxx', 'hostname': '', 'type_id': '0', 'vendor': '', 'model': 'McAfee Event Receiver', 'tz_id': '', 'date_order': '', 'port': '', 'syslog_tls': '', 'client_groups': 'xxx', 'zone_name': 'xxx', 'zone_id': 1, 'client': False, 'parent_name': 'Local ESM', 'parent_id': 'xxx', 'last_time': '21-07-2020 01:58:06'}
```

Meanwhile via the GUI at the same time, in the Receiver Properties -> Device Log for that ERC, I see a `Last Event` time of `2020/07/22 06:56:22`. Both the GUI account and the host running the script are in UTC time. Back in v10.4.0.x, these values would have been the same. 

_As an aside, it looks like individual data source times are being reported correctly so I imagine a lot of users of this tool won't be impacted._ I have a ticket open with McAfee; apparently it was a 'known issue' that was supposed to be resolved in patch 13. 

Anyway, just in-case anyone else runs into this. I'll update once the ticket is resolved one way or another.

**Comment by indistinctTalk** [comment]
McAfee have closed as `wontfix` and recommend using the `qryGetDeviceLastAlertTime` endpoint instead.
